src/net_processing.cpp

```diff
@@ -2980,8 +2990,18 @@ void ProcessMessage(
         else if (state.GetResult() == TxValidationResult::TX_MISSING_INPUTS)
         {
             bool fRejectedParents = false; // It may be the case that the orphans parents have all been rejected
+
+            // Deduplicate parent txids, so that we don't have to loop over
+            // the same parent txid more than once down below.
+            std::vector<uint256> unique_parents;
+            unique_parents.reserve(tx.vin.size());
```

Comment:
I think `parents` would be a better name here. They only become unique later once you deduplicate them.

Reply:
Added a comment instead, since the first thing we do is deduplicate, and then we use this as a list of unique_parents later on.